test/unit/common.test.js

```diff
@@ -20,10 +20,18 @@ const { getSdkHeaders } = common;
 
 describe('Tests of Common Library', () => {
   describe('getSdkHeaders', () => {
-    test('should return correct User-Agent header', () => {
-      const headers = getSdkHeaders('service1', 'v1', 'operation1');
+    test('should return correct X-IBMCloud-SDK-Analytics header', () => {
```

Comment:
minor: this is testing both analytics _and_ user agent headers

Reply:
Right! I modified it in [e8a8f75](https://github.com/IBM/cloudant-node-sdk/pull/721/commits/e8a8f75cd320c6b1b9c699fac1a27a47ec534e82)